CalibMuon/DTCalibration/python/ALCARECODtCalib_cff.py

```diff
@@ -10,7 +10,7 @@
 #ALCARECODtCalibHLTFilter.andOr = True ## choose logical OR between Triggerbits
 #ALCARECODtCalibHLTFilter.HLTPaths = ['HLT_L1MuOpen*', 'HLT_L1Mu*']
 ALCARECODtCalibHLTFilter.throw = False ## dont throw on unknown path names
-ALCARECODtCalibHLTFilter.eventSetupPathsKey = 'DtCalib'
+ALCARECODtCalibHLTFilter.eventSetupPathsKey = 'MuAlcaDtCalibMu'
```

Comment:
why this eventSetupPathsKey has been changed? What we have always is DtCalib. This is basically the reason for the failure of relvals because this key is part of a tag which contains all the keys associated to AlCaRecoTriggerBits. Please revert this change

Reply:
I am waiting for a reply from the other contributors.